src/pykeen/models/base.py

```diff
@@ -112,18 +115,18 @@ def __init__(
         else:
             self.loss = loss_resolver.make(loss, pos_kwargs=loss_kwargs)
 
-        self.use_inverse_triples = triples_factory.create_inverse_triples
+        self.use_inverse_triples = create_inverse_triples
```

Comment:
Maybe we should agree on some consistent naming

Reply:
let's do that in a follow-up PR, since the diff is already so big